src/build/env.py

```diff
@@ -11,6 +11,10 @@
     from typing import Dict, Optional, Iterable, Type
 
 
+if sys.version_info[0] == 2:  # pragma: no cover
+    FileExistsError = OSError
```

Comment:
This is kinda suprising way to handle this, though I suppose is correct 👍

Reply:
Well, I tried to unbundle from the rest of the code.